ruby/basics/conditional_logic.md

```diff
@@ -1,43 +1,246 @@
+
+```ruby
+if room_tidy == true
+  ask mother if I can play video games
+end
+```
+There must be a motherboard joke in there somewhere. Answers on a postcard!
+
+Any conditional statement will always have an expression that evaluates to `true` or `false` and on that basis the computer will decide whether to action the code that follows. If it's true then the code will be processed; false, the code will be skipped, and you can provide some other code to run instead. There can even be several conditional statements on one line although keep in mind that too many can make code look cluttered.
+
+You'll write a lot of conditional statements on your road to programmer stardom and although they are pretty simply at heart, they're are a big source of bugs in your code when something isn't working as expected so make sure you understand the logic behind the expression being evaluated so you can step through it if you need to.
+
 ## Learning outcomes
 *Look through these now and then use them to test yourself after doing the assignment*
 
-what the student is expected to know or be able to do by the end of this lesson
-
-* Learning outcome 1
-* Learning outcome 2
-* Learning outcome 3
+* Conditionals and Flow Control
+    * What is a "boolean"?
+    * What are "truthy" values?
+    * Are nil, 0, "0", "", 1, [], {} and -1 considered true or false?
+    * When do you use elsif?
+    * When do you use unless?
+    * What does <=> do?
+    * Why might you define your own <=> method?
+    * What do || and && and ! do?
+    * What is returned by puts("woah") || true?
+    * What is ||=?
+    * What is the ternary operator?
+    * When should you use a case statement?
 
 ## Truthy and Falsy in Ruby
-- Brief explanation what ruby condiders to be true and what ruby considers to be false. 
-- The only two things in ruby which are false is nil and false
+
+You already know that conditional statements check expressions for a true or false value so it follows that you should understand what Ruby considers to be true or false. In typical Ruby fashion it's very simple. The only false values in ruby are the values `nil` and `false` itself. That's it! Everything else is considered true. Even the string `"false"` is true in conditional expressions! If you have experience with other programming languages you might be familiar with the number 0 or an empty string "" being equivalent to false, this isn't the case with Ruby so be careful when writing those expressions otherwise you'll have more bugs than a decomposing body.
 
 ## Basic Conditional Statement
-- How to create a single branch if statement
-- single line if statements
+The simplest way to control the flow of your code using conditions is with the `if` statement.
+
+The general syntax is
+
+```ruby
+if (statement_to_be_evaluated == true)
+  do something awesome...
```

Comment:
do is another ruby keyword that might cause confusion here, to avoid that I think we should comment it out `#do something awesome...`